playwright/test/contexts.ts

```diff
@@ -197,7 +197,7 @@ const concept = async (
 
 const articleWithMockSiblings = async (
   id: string,
-  response: Record<string, any>,
+  response: Record<string, unknown>,
```

Comment:
An `unexpected any` prevented TS from building after merging `main` so I've changed this file accordingly.